build/NI.CSharp.Analyzers.nuspec

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <id>NI.CSharp.Analyzers</id>
+    <version>0.1.0</version>
```

Comment:
Would be nice to auto-version this on submit. Something like GitVersion could help (and there are many alternatives!): https://github.com/GitTools/GitVersion